mon-pix/app/components/signin-form.js

```diff
@@ -27,6 +28,11 @@ export default class SigninForm extends Component {
     return this.url.isFrenchDomainExtension;
   }
 
+  get displayFwbButton() {
+    const identityProviderId = 'fwb';
+    return this.oidcIdentityProviders.isIdentityProviderActivated(identityProviderId);
+  }
+
```

Comment:
Comme `fwb` est une constante qui ne variera jamais dans le temps, mieux vaut l'instancier une seule fois.

```suggestion
const OIDC_PROVIDER_FWB_ID = 'fwb';

  get displayFwbButton() {
    return this.oidcIdentityProviders.isIdentityProviderActivated(OIDC_PROVIDER_FWB_ID);
  }

```

Reply:
En fait, mieux vaut tout mettre toute l'implémentation dans le service `@service oidcIdentityProviders` et comme ça pas besoin de définir de constante `fwb` dans le code la signin-form : 

```suggestion
  get displayFwbButton() {
    return this.oidcIdentityProviders.isFwbActivated();
  }

```